client/signup/step-wrapper/index.jsx

```diff
@@ -280,4 +284,4 @@ export default connect( ( state, ownProps ) => {
 		backUrl,
 		userLoggedIn: isUserLoggedIn( state ),
 	};
-} )( localize( StepWrapper ) );
+} )( localize( withGeoLocation( StepWrapper ) ) );
```

Comment:
I had thought about this but then chose to have the geo check in the step button. It's not an elegant solution, but with this, we'll have a call to /geo in every step of every /start flow, even the ones unrelated to DIFM.